src/pages/Thanks/ThanksPage.vue

```diff
@@ -1,11 +1,11 @@
 <template>
-	<www-page data-testid="thanks-page">
+	<www-page data-testid="thanks-page" :class="{'tw-bg-eco-green-1': !showOldThanksPage && !isOnlyDonation}">
 		<template v-if="isOnlyDonation">
 			<thanks-page-donation-only
 				:monthly-donation-amount="monthlyDonationAmount"
 			/>
 		</template>
-		<template v-else>
+		<template v-else-if="showOldThanksPage">
```

Comment:
Are we sure this won't hide some behavior that we want? This part of the template covers a number of different thanks page variations.

Reply:
It seems like this is what is being requested by product. However, I agree that it doesn't feel right. @christian14b I think it would be good to also put this new thanks page behind a feature flag so that we can safely rollout the code changes to production while we test the different checkout scenarios in development and review them with product.